ssl4rs/data/datamodules/disa.py

```diff
@@ -246,6 +251,15 @@ def convert_deeplake_tensors_to_pytorch_tensors(
         batch[tname] = tval
     return batch
 
+# todo: maybe change so this is not inplace?
+def convert_4_band_to_3_band(
+    batch: ssl4rs.data.BatchDictType,
+) -> ssl4rs.data.BatchDictType:
+    """Transform converts RGB+NIR into a 3-band composite suitable for pretrained backbones"""
+    img_data = batch['image_data']
+    img_transform = ssl4rs.data.transforms.composite_bands.Convert4BandTo3Band()
+    batch['image_data'] = img_transform(img_data)
+    return batch
```

Comment:
there are some formatting issues in this function, make sure to run the linter ("make check") to catch/fix them